test/test_coordgen.cpp

```diff
@@ -15,6 +15,9 @@
 #include "maeparser/MaeConstants.hpp"
 #include "maeparser/Reader.hpp"
 
+// The sketcherMinimizerFragment class is not exposed in the library
+#include "../sketcherMinimizerFragment.cpp"
```

Comment:
Won't this cause a double definition of the class?

I've seen weird issues in Maestro tests when they do this.

I guess I'd rather expose some sort of cleanuper.

Reply:
We'd have to expose the whole class to get access to the destructor.